### DISCUSSION TITLE
Is it possible to make react-aria to vanilla-aria?

### DISCUSSION THREAD

**Discussion by XMethues** [adobe/react-spectrum#3823] [General]
First, I really like react-aria and react-stately. 
As more frameworks we have now(svelte,vue,solidjs....). If we could make the library cross framework. It would be awesome. 

I tried to use xstate as the state management to create useHover in svelte, but it was awkward. So, I think the if there is a vanilla state management. Vanilla aria can become true.
btw,I see there is solid-aria.

**Comment by snowystinger** [comment]
Dupe discussion here https://github.com/adobe/react-spectrum/issues/1329
Unfortunately it hasn't been in priorities for the team since Adobe doesn't use those other frameworks, but we'd be open to a contribution handling it.
